theano/scan_module/tests/test_scan.py

```diff
@@ -3173,9 +3173,9 @@ def lambda_fn(h, W1, W2):
                     if isinstance(x.op, theano.tensor.Elemwise)]) == 0
 
     def test_alloc_inputs2(self):
-        raise KnownFailureTest((
-            "This tests depends on an optimization for scan "
-            "that has not been implemented yet."))
+        raise SkipTest("Not yet implemented")
```

Comment:
You removed the old documentation at many places. Keep it everywhere except if you are sure the new one is better.